docs/source/installation/for_dev.rst

```diff
@@ -0,0 +1,126 @@
+Manim Installation For Developers
+=================================
+
+	This documentation is for developers who wats to contribute to ``ManimCommunity/manim``.
+
+Installing Poetry
+*****************
+
+Poetry can be easily installed in any OS by just running the below command.
+
+.. code-block:: bash
+	
+	curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python
+
+
+If you are a Windows developer and want to use PowerShell, you can use the below command.
+
+.. code-block:: powershell
+	
+	(Invoke-WebRequest -Uri https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py -UseBasicParsing).Content | python
+
+It will add it to path automatically.
+
+.. code-block:: bash
+
+	poetry --version
+
+
+.. note:: You may need to restart your shell in order for the changes to take effect.
+
+See the `docs on installation python poetry for more information
+<https://python-poetry.org/docs/>`_.
+
+Installing System Dependencies
+******************************
+
+This section had to followed as in :ref:`installing-manim`.
+
+.. important:: Windows Users can Ignore Pycairo Installation.
+
+
+Additional `git` has to be installed. For instructions see `Documentation
+<https://git-scm.com/>`_.
+
+.. _Install Manim Poetry:
+Installing Manim using Poetry
+*****************************
+
+1.  First, clone the Manim repo locally using git.
+
+    .. code-block:: bash
+		
+		git clone https://github.com/ManimCommunity/manim.git
+
+    or
+
+    .. code-block:: bash
+		
+		git clone git@github.com:ManimCommunity/manim.git
+
+2.  Open a Terminal/Powershell/Command Prompt and cd into the cloned directory.
+
+    .. code-block:: bash
+		
+		cd path/to/manim
+    
+
+    .. note:: This path should contain a file called `pyproject.toml` if it doesn't contain it, you would need to go a level up.
+
+3.  Use the below command to install dependencies. This will use default python version installed.
```

Comment:
```suggestion
3.  Use the below command to install dependencies. This will use the default python version installed.
```

Question: The previous section says that the instructions in :ref:`installing-manim` have to be followed in order to install dependencies. But now this section says that this command will install the dependencies. This is contradictory, no?